gensrc/proto/lake_types.proto

```diff
@@ -86,6 +86,10 @@ message TabletMetadataPB {
     optional PersistentIndexTypePB persistent_index_type = 12;
 }
 
+enum TabletMetaTypePB {
```

Comment:
Perhaps we can determine which contents of metadata need modification through Protobuf's has_xxx(), just like how we currently judge which OpXXX to execute in the txn log?

Reply:
That's OK, more elegant